### ISSUE TITLE
Re-running previously demultiplexed samples

### ISSUE THREAD

**Issue by biobenkj** [mjmlab/pyinseq#31]
Add a couple new options (--checkpoint or --restartdir?) to be able to go into the experiment folder and grab the samples.yml file for paths to demultiplexed data

**Comment by mandel01** [comment]
I think this is a great idea and one that we hack all of the time in the lab to compare samples from different sequencing runs. There is an old `--rerun` argument; when I rewrote the pipeline I didn't port over the code but figured we'd find a better way here. I wasn't sure we had the infrastructure to do this yet. Will send more thoughts on this later.

**Comment by mandel01** [comment]
Correction that was the `--nobarcodes` argument that is currently in `pyinseq.py` but unused. The reason I got that confused is no.2 below...

Two issues to consider here...

1. For reproducibility, if the user uses samples in experiment01 in experiment02, they need to know not to delete the experiment01 directory or they would lose an intermediate step. Could leave that up to the user. Could copy over the demultiplexed samples. Or perhaps there is a more clever solution.


2. People get HiSeq results in one of (at least) three formats. This might be a good place to think about incorporating that issue as well. Or I can start it as a separate issue.

- Not demultiplexed at all
- Demultiplexed (in which case we start at the same point that we want for grabbing samples that we've already demultiplexed)
- Demultiplexed with 5' 4-bp barcodes removed (in which case the barcode length should be set to 0 and there may need to be some refactoring of chromosome sequence extraction)
